spec/fixtures/files/sortable/tailorings_controller.yaml

```diff
@@ -0,0 +1,20 @@
+:entities:
+  - :factory: :v2_tailoring
+    :os_minor_version: 0
+    :policy_id: ${policy_id}
+    :ref_id: ${policy.ref_id}
+
+  - :factory: :v2_tailoring
+    :os_minor_version: 1
+    :policy_id: ${policy_id}
+    :ref_id: ${policy.ref_id}
+
```

Comment:
Another one with `os_minor_version: 1` that has to be reflected in the sorting to test if the secondary sorting works well using IDs.

Reply:
There can be only one tailoring for each `os_minor_version` under a policy.